### ISSUE TITLE
Forms are not showing web responsive

### ISSUE THREAD

**Issue by TapAloha** [yikesinc/yikes-inc-easy-mailchimp-extender#477]
Hi Yikes, 

The forms I've implemented using your plug-in are not rendering well on mobile. How is this fixed? If it can not be fixed I would please like a refund. 

Thank you. :)

![screen shot 2016-03-23 at 11 56 59 am](https://cloud.githubusercontent.com/assets/17835533/13997242/685d2a44-f0ee-11e5-9241-6641fb4df4ff.png)

**Comment by TheTracyL** [comment]
Hi there,

Can you please provide a link to your site?

Thank you!
-Tracy

**Comment by TapAloha** [comment]
Hi Tracy,

Yes, it is AlohaIke.com.

We're trying to get it up fast and came across your plug-in to help the
process.

Thanks.

Best,
Adriana

On Wed, Mar 23, 2016 at 11:59 AM, Tracy Levesque <notifications@github.com>
wrote:

> Hi there,
>
> Can you please provide a link to your site?
>
> Thank you!
> -Tracy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-200496325>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Hello Adriana,

Can you please pop this code in to the [Custom CSS](https://yikesplugins.com/support/knowledge-base/form-customizer-for-easy-forms-for-mailchimp/#custom-css) portion of the Customizer:

```CSS
@media screen and (max-width: 768px) {
	.yikes-easy-mc-form label.label-inline {
	    float: none;
	    height: 68px;
	    margin: 0;
	    width: 100%;
	}
}
```

Let me know if that works.

Thank you!
-Tracy

**Comment by TapAloha** [comment]
Hi Tracy,

Nope still seeing the same thing.

We're also having problems with the duplicate function.

When we use it the styling is not duplicated and sometimes will mess up the
styling of the form we are duplicating. This is way the form at the bottom
of the page does not match the form at the top of the page.

We appreciate your help with both issues.

Thanks Much,
Adriana

On Wed, Mar 23, 2016 at 12:15 PM, Tracy Levesque <notifications@github.com>
wrote:

> Hello Adriana,
>
> Can you please pop this code in to the Custom CSS
> <https://yikesplugins.com/support/knowledge-base/form-customizer-for-easy-forms-for-mailchimp/#custom-css>
> portion of the Customizer:
>
> @media screen and (max-width: 768px) {
>     .yikes-easy-mc-form label.label-inline {
>         float: none;
>         height: 68px;
>         margin: 0;
>         width: 100%;
>     }
> }
>
> Let me know if that works.
>
> Thank you!
> -Tracy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-200501494>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Ok, just a couple tweaks needed:

```CSS
@media screen and (max-width: 768px) {
    .yikes-easy-mc-form label.label-inline {
        float: none  !important;
        height: 68px;
        margin: 0;
        width: 100%  !important;
    }
}
```

Please try that.

Thank you!
-Tracy

**Comment by TapAloha** [comment]
That made it responsive but took away our inline, duplicated the form field
in multiple places, and changed the styling. :(

Thanks,
Adriana

On Wed, Mar 23, 2016 at 12:25 PM, Tracy Levesque <notifications@github.com>
wrote:

> Ok, just a couple tweaks needed:
>
> @media screen and (max-width: 768px) {
>     .yikes-easy-mc-form label.label-inline {
>         float: none  !important;
>         height: 68px;
>         margin: 0;
>         width: 100%  !important;
>     }
> }
>
> Please try that.
>
> Thank you!
> -Tracy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-200505308>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Ok, this will make the styles only apply to that form on the home page and no others. Also, it will only stack from iPad portrait and narrower. When I run it through the Responsinator, that is when the layout starts to look bad inline.

```CSS
@media screen and (max-width: 768px) {
    #yikes-mailchimp-container-5 .yikes-easy-mc-form label.label-inline {
        float: none  !important;
        height: 68px;
        margin: 0;
        width: 100%  !important;
    }
}
```

**Comment by TapAloha** [comment]
Woo! That worked!

Super impressed with how fast the support was.

Should I submit another ticket for the duplication challenges we're having?

Thanks so much!

Best,
Adriana

On Wed, Mar 23, 2016 at 12:33 PM, Tracy Levesque <notifications@github.com>
wrote:

> Ok, this will make the styles only apply to that form on the home page and
> no others. Also, it will only stack from iPad portrait and narrower. When I
> run it through the Responsinator, that is when the layout starts to look
> bad inline.
>
> @media screen and (max-width: 768px) {
>     #yikes-mailchimp-container-5 .yikes-easy-mc-form label.label-inline {
>         float: none  !important;
>         height: 68px;
>         margin: 0;
>         width: 100%  !important;
>     }
> }
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-200509442>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Woot!

As for the duplicate forms, each form is treated independently and you'll have to style each one separately. Is that the problem? Are you experiencing having to do each one, or are you finding the styles to be crossing over?

Let me know :-)

Thank you!
-Tracy

**Comment by TapAloha** [comment]
I'm finding that the styling does not cross over.

In addition, duplicating will sometimes change the styling in the original
form field. The styling that has consistently been changed in the original
form field is the the inline formatting even though the inline is selected.

Thanks much!

Best,
Adriana

On Wed, Mar 23, 2016 at 12:47 PM, Tracy Levesque <notifications@github.com>
wrote:

> Woot!
>
> As for the duplicate forms, each form is treated independently and you'll
> have to style each one separately. Is that the problem? Are you
> experiencing having to do each one, or are you finding the styles to be
> crossing over?
>
> Let me know :-)
>
> Thank you!
> -Tracy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-200515648>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Ok, yes, each form needs to be styled independently of each other. Do you see the styling changed after you save and back out of the Customizer? 

If you have certain styles you want shared by all forms, I can give you some Custom CSS to save you time.

**Comment by TapAloha** [comment]
Please see below... :)

Super awesome support btw! Totally gonna recommend.



On Wed, Mar 23, 2016 at 12:53 PM, Tracy Levesque <notifications@github.com>
wrote:

> Ok, yes, each form needs to be styled independently of each other. Do you
> see the styling changed after you save and back out of the Customizer?
>

Yes. For example, the form field at the bottom of the home page should look
like the top form field but ended up looking like that we it was
"duplicated".

> If you have certain styles you want shared by all forms, I can give you
> some Custom CSS to save you time.
>

That would be awesome!!!  Right now the styling we prefer is the one at the
top of the page.

While we have no boarder on the 'Get Ike' button it still looks like there
is a thin blue line. We'd like to keep the the boarder radius but have the
boarder outline and hover styling of the button on the call-to-actions on
the Google Analytics page
https://www.google.com/analytics/#?modal_active=none.




> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-200517278>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Hi Adriana,

Here is the styling for the buttons. Your theme had a button border style that was overriding our Customizer.

```CSS
.yikes-easy-mc-form .yikes-easy-mc-submit-button {
	background-color: #00c853;
	background: #00c853;
	border-color: #00c853;
}

.yikes-easy-mc-form .yikes-easy-mc-submit-button:hover, .yikes-easy-mc-form .yikes-easy-mc-submit-button:focus {
	background-color: #43a047;
	background: #43a047;
	border-color: #43a047;
}
```

We may have to add some importants some some of those styles, but lets take a look first.

For the form on the bottom of the page you will need to add placeholders and hide the labels. Then I can give you some universal placeholder styles.

Thank you!
-Tracy

**Comment by TheTracyL** [comment]
Edited to add `border-radius`.

**Comment by TapAloha** [comment]
I added the CSS you provided to the bottom form field but all the styling
did not translate. Do I still need to manually go in and make other
adjustments after I use the "Duplicate" function?

Thanks!


On Wed, Mar 23, 2016 at 1:21 PM, Tracy Levesque <notifications@github.com>
wrote:

> Edited to add border-radius.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-200527496>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Hello.

I just took a look and the buttons seem to have the styles you want.

What is missing, exactly?

Thank you!
-Tracy

**Comment by TapAloha** [comment]
Hi Tracy,

The sizing of the form fields, the button size, and the form titles in the
form fields not on top. None of these styling attributes duplicated using
the duplicate function.

Thanks again for your help.

Best,
Adriana

On Thu, Mar 24, 2016 at 7:26 AM, Tracy Levesque <notifications@github.com>
wrote:

> Hello.
>
> I just took a look and the buttons seem to have the styles you want.
>
> What is missing, exactly?
>
> Thank you!
> -Tracy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-200861491>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Hi Adriana,

Like I mentioned previously, each form is styled independently of each other, the styles don't duplicate.

I need you to do this step I mentioned above:
> For the form on the bottom of the page you will need to add placeholders and hide the labels. 

Once that is done I can style the fields.

Thank you!
-Tracy

**Comment by TheTracyL** [comment]
Hi Adriana,

Just checking in on this. Were you able to do the placeholder step?

Thank you!
-Tracy

**Comment by TapAloha** [comment]
Hi Tracy,

Thanks for the follow up! Much appreciated. I lost the placeholder step in
the thread. We're still having the same problem. Can you please share the
details again?



On Tue, Apr 5, 2016 at 10:11 AM, Tracy Levesque <notifications@github.com>
wrote:

> Hi Adriana,
>
> Just checking in on this. Were you able to do the placeholder step?
>
> Thank you!
> -Tracy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-205902248>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Sure.

> For the form on the bottom of the page you will need to add placeholders and hide the labels.

To add placeholder text and hide the field labels [follow the instructions from this blog post](https://yikesplugins.com/the-ultimate-guide-to-translating-mailchimp-forms-on-wordpress-sites/) at the bottom under 
**Bonus Step – Translate the Placeholder Text**

Thank you!
-Tracy

**Comment by TapAloha** [comment]
Okay. Done.

I've tried to put in the CSS to make it web responsive. I may have a typo.
Can you please share it again. Thanks. :)

On Tue, Apr 5, 2016 at 10:55 AM, Tracy Levesque <notifications@github.com>
wrote:

> Sure.
>
> For the form on the bottom of the page you will need to add placeholders
> and hide the labels.
>
> To add placeholder text and hide the field labels follow the instructions
> from this blog post
> <https://yikesplugins.com/the-ultimate-guide-to-translating-mailchimp-forms-on-wordpress-sites/>
> at the bottom under
> *Bonus Step – Translate the Placeholder Text*
>
> Thank you!
> -Tracy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-205919361>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
Here you go!

```CSS
@media screen and (max-width: 768px) {
    #yikes-mailchimp-container-6 .yikes-easy-mc-form label.label-inline {
        float: none  !important;
        height: 68px;
        margin: 0;
        width: 100%  !important;
    }
}
```

**Comment by TapAloha** [comment]
Hi Tracy,

Thank you. I went through the same process for this page:
http://alohaike.com/get-ike/.

And, it is still not showing up with the same Web responsiveness as the
others. Thoughts?

Best,
Adriana

On Wed, Apr 6, 2016 at 1:04 PM, Tracy Levesque <notifications@github.com>
wrote:

> Here you go!
>
> @media screen and (max-width: 768px) {
>     #yikes-mailchimp-container-6 .yikes-easy-mc-form label.label-inline {
>         float: none  !important;
>         height: 68px;
>         margin: 0;
>         width: 100%  !important;
>     }
> }
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yikesinc/yikes-inc-easy-mailchimp-extender/issues/477#issuecomment-206534924>
>



-- 
Adriana Herrera
Founder and CEO

Email: adriana@tapaloha.com
Mobile: 808.990.1943

TapAloha, http://tapaloha.com/
Improve Performance Across Your Site, App, and Marketing

**Comment by TheTracyL** [comment]
That form has a different ID (8). You can use this:

```CSS
@media screen and (max-width: 768px) {
    #yikes-mailchimp-container-6 .yikes-easy-mc-form label.label-inline,
    #yikes-mailchimp-container-8 .yikes-easy-mc-form label.label-inline  {
        float: none  !important;
        height: 68px;
        margin: 0;
        width: 100%  !important;
    }
}
```

Do you have more forms set up? I added the ID so the styles didn't universally apply to every form since you have them in different areas.

Thank you!
-Tracy

**Comment by TheTracyL** [comment]
Hi Adriana,

Are you all set now?

Thank you!
-Tracy
